apps/settings/locales/settings.en-US.properties

```diff
@@ -1233,7 +1233,8 @@ hud-security=Security Issues
 hud-performance=Performance
 hud-reflows=Reflows
 hud-jank=Jank
-jank-threshold=Jank Threshold
+jank-threshold=Jank Threshold (ms)
```

Comment:
Since you need to change this ID, please make it consistent with the other IDs (i.e. something like `hud-jank-threshold`).